packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.ts

```diff
@@ -130,6 +130,54 @@ export class ERC1155Standard {
     });
   };
 
+  /**
+   * Query for symbol for a given asset.
+   *
+   * @param address - ERC1155 asset contract address.
+   * @returns Promise resolving to the 'symbol'.
+   */
+  getAssetSymbol = async (address: string): Promise<string> => {
+    const contract = new Contract(
+      address,
+      [
+        {
+          inputs: [],
+          name: 'symbol',
+          outputs: [{ name: '_symbol', type: 'string' }],
+          stateMutability: 'view',
+          type: 'function',
+          payable: false,
+        },
+      ],
+      this.provider,
+    );
+    return contract.symbol();
+  };
+
+  /**
+   * Query for name for a given asset.
+   *
+   * @param address - ERC1155 asset contract address.
+   * @returns Promise resolving to the 'name'.
+   */
+  getAssetName = async (address: string): Promise<string> => {
```

Comment:
These functions are declared `async` but they do not `await`.  I'm wondering whether they need the async, or if they could just return the promise.